src/the_program_tunserver.cpp

```diff
@@ -206,6 +206,10 @@ std::tuple<bool,int> c_the_program_tunserver::base_options_commands_run() {
 }
 
 int c_the_program_tunserver::main_execution() {
+	_mark("Main execution of the old-loop");
+	_warn("Remember, that this old-loop code is NOT secured as new-loop code, e.g. is not droping CAP/root privileges!");
+	{ using namespace std::chrono_literals;	std::this_thread::sleep_for(1s); }
```

Comment:
So user can see the above warning clearly.
1 second seems not too long and still visible.

Documented